app/models/automated_checks/dqt_report_csv_to_records.rb

```diff
@@ -30,5 +31,9 @@ def parse_date(date)
     rescue
       nil
     end
+
+    def extract_surname(fullname)
+      fullname&.split(" ")&.last
```

Comment:
I tried to do `fullname && fullname.split(" ").last` to avoid all the lonely operators but Standard complained. Any suggestions to make the neater are welcome :)

Reply:
Maybe this?
```suggestion
      (fullname || "").split(" ").last
```

Reply:
Thanks!